src/Raven.Client/Documents/TimeSeries/TimeSeriesOperations.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Operations.TimeSeries;
+using Raven.Client.Documents.Session.TimeSeries;
+
+namespace Raven.Client.Documents.TimeSeries
+{
+    public class TimeSeriesOperations
```

Comment:
We need an async version as well

Reply:
it is all `async`, we need sync as well?

Reply:
I'm wondering if we need it at all. All of this can be handled inside `ConfigureTimeSeriesValueNamesOperation`.